libs/deal-ticket/src/components/deal-ticket-validation/use-order-validation.tsx

```diff
@@ -308,5 +338,5 @@ export const useOrderValidation = ({
     isInvalidOrderMargin,
   ]);
 
-  return { message, isDisabled };
+  return { message, isDisabled, section: section as DealTicketSection };
```

Comment:
suggestion (non blocking): The type cast here hides the fact that useMemo can return `string`. I think a better solution would be to have a specific type for 'none' state. IE:. `constants.DEAL_TICKET_SECTION_TYPE_NONE`

Reply:
casting could be removed only in case when useMemo got explicitly returned types